css/02-elements.css

```diff
@@ -12,7 +12,9 @@ button, input, select, optgroup, textarea {
 }
 
 h1, h2, h3, h4, h5, h6 {
-	font-family: 'Arial Black', Gadget, sans-serif;
```

Comment:
Arial is a system font so it shouldn't need a fallback, only sans-serif is needed here e.g. `font-family: 'Arial', sans-serif;` (unless Gadget is a system font I haven't heard of before, which is possible).

Reply:
Apparently it's an Apple system font, but it's not a better match and we don't need it. Took it out and will push the change shortly...